ts/components/wallet/paymentSummary/PaymentSummaryComponent.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * this component will display the transaction details if no updates on the amount are not identified
+ */
+import { H1, H3, Right, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import I18n from "../../../i18n";
+import variables from "../../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+  amount: string;
+  expireDate: string;
```

Comment:
this should be a `Date`

Reply:
you can use https://date-fns.org/ for parsing - cc @gunzip

Reply:
What concerns the value, it seems to not being an available info. Maybe more details will come from sia